internal/accounts/create-interactive.go

```diff
@@ -259,6 +263,7 @@ func (l *lilicoAccount) create(network string) (flowsdk.Identifier, error) {
 	request.Header.Add("Authorization", accountToken)
 
 	client := &http.Client{
+		Timeout: time.Second * 20,
```

Comment:
Should we add a timeout on `getAccountCreationResult` also?

Reply:
Is that in flowkit @jribbink ?

Reply:
Yeah not sure.  If it's not easy, don't bother... it's not that important